packages/ubuntu_bootstrap/test/install/install_model_test.dart

```diff
@@ -134,10 +133,9 @@ void main() async {
 
     final session = MockSessionService();
     final config = MockConfigService();
+    when(config.provisioningMode).thenReturn(Future<ProvisioningMode>.value(ProvisioningMode.standard));
```

Comment:
Could you use `thenAnswer` here? See [docs](https://pub.dev/packages/mockito#a-quick-word-on-async-stubbing).

Reply:
You don't need a `Future.value` then, but instead use
```dart
when(config.provisioningMode).thenAnswer((_) async => ProvisioningMode.standard));
```

Reply:
Good point! Changed.